packages/backend/src/functions/httpTrigger.ts

```diff
@@ -400,3 +402,8 @@ app.get("getStatistics", {
     route: 'statistics',
     handler: getStatistics
 })
+
+
+

```

Comment:
Are you able to create a new endpoint here that we can use to test the email client is working in Azure?